app/javascript/components/Home/Nav.tsx

```diff
@@ -1,115 +0,0 @@
-import * as React from "react";
```

Comment:
Are we sure this isn't used? I see a bunch of result for NavLink in the codebase (I'm guessing there was an alternative one)